osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -107,24 +126,35 @@ func addPackageRemovalTest(pkgTestSetup []*packageManagementTestSetup) []*packag
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
 		var assign *osconfigpb.Assignment
-		var instaneName, ss, vs string
+		var instanceName, image, ss, vs string
 
 		switch pkgManager {
 		case "apt":
-			instaneName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
+			image = debianImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
```

Comment:
path.Base, and move this down to packageManagementTestSetup

Reply:
changed